feature/aft/aft_base/otg_tests/aft_prefixcounters/README.md

```diff
@@ -0,0 +1,192 @@
+# AFT-2.1: AFTs Prefix Counters
+
+## Summary
+
+IPv4/IPv6 prefix counters
+
+## Procedure
+
+Configure DUT:port1 for IS-IS session with ATE:port1
+*   IS-IS must be level 2 only with wide metric.
+*   IS-IS must be point to point.
+*   Send 1000 ipv4 and 1000 ipv6 IS-IS prefixes from ATE:port1 to DUT:port1.
+*   IS-IS must be up and running, prefixes must be in RIB and FIB.
+
+Establish eBGP sessions between ATE:port1 and DUT:port1.
+*   Configure eBGP over the interface ip.
+*   Advertise 1000 ipv4,ipv6 prefixes from ATE port1 observe received prefixes at DUT.
+*   Validate total number of entries of AFT for IPv4 and IPv6.
+*   Each prefix must have 1 next hop pointing to ATE port1.
+
+Verifications
+
+*   BGP routes advertised from ATE:port1 must have 1 nexthop.
+*   IS-IS routes advertised from ATE:port1 must have one next hop.
+
+
+## AFT-2.1.1: AFT Prefix Counters ipv4 packets forwarded, IS-IS route.
```

Comment:
Minor, optional comment-- since you're sending packets and then waiting a bunch for the packets to be sent, to minimize runtime of the test, you can group the checks (check packets and octets in the same test), and send a fixed number of packets, instead of waiting for a fixed amount of time.

Then, you can subscribe to the data and make sure it updates within a prescribed amount of time after the packets have been sent.

Reply:
ok

Reply:
done

Reply:
Resending the same packet flows to count packets vs. octets  is making the readme longer and more difficult to read and extending the test run time which is expensive and not adding value to the functionality of the test.  The device counts octets and packets for every flow.

That said, it is ok to have separate validations for ipv4 and ipv6 and packets and octets.  But there is no reason to resend the same flow over and over again.  

 Please refactor this to only send the ipv4 and ipv6 flows once each.  (separate v4 and v6 flows is ok, they don't need to be mixed).  You can then break out the validations however your choose.

Reply:
added for ipv4 packets and octets together